ci-operator/step-registry/cucushift/installer/check/vsphere/zone/cucushift-installer-check-vsphere-zone-commands.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+source "${SHARED_DIR}/govc.sh"
+source "${SHARED_DIR}/vsphere_context.sh"
+INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
+set -x
+
+check_result=0
+readarray -t zones_name_from_config < <(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains[*].name")
+infra_id=$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)
+
+function check_rhcos() {
+    local check_result=0 fd_folder fd_datacenter fd_region fd_zone
+    # shellcheck disable=SC2207
+    for name in "${zones_name_from_config[@]}"; do
+        fd_folder=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains.(name==${name}).topology.folder")
+	fd_datacenter=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains.(name==${name}).topology.datacenter")
+        [[ -z "${fd_datacenter}" ]] && fd_datacenter=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.datacenter")
+	fd_region=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains.(name==${name}).region")
+	fd_zone=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains.(name==${name}).zone")
+	[[ -z "${fd_folder}" ]] && fd_folder="/${fd_datacenter}/vm/${infra_id}"
+        if [[ $(govc ls ${fd_folder} | wc -l ) -gt 0 ]]; then
+	    echo "INFO: folder ${fd_folder} created successful"
+	    if [[ $(govc ls ${fd_folder}/${infra_id}-rhcos-${fd_region}-${fd_zone} | wc -l ) -gt 0 ]]; then
```

Comment:
The same comments as the above, suggest to use `govc` command to run the check directly to avoid `wc -l`

Reply:
ok.will update